Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -283,6 +283,7 @@ MediaPlayerPrivateGStreamerBase::MediaPlayerPrivateGStreamerBase(MediaPlayer* pl
     , m_isEndReached(false)
     , m_usingFallbackVideoSink(false)
     , m_drawTimer(RunLoop::main(), this, &MediaPlayerPrivateGStreamerBase::repaint)
+    , m_initDataCount(0)
```

Comment:
Another thing, this is defined under `USE(OPENCDM)` in the header, but not here, so if you build with opencdm support, it will fail. Worth doing that as a check to make sure there's not other places I guess